examples/nivisa_dmm_measurement/measurement.py

```diff
@@ -109,8 +133,8 @@ def measure(
             if not session_info.session_exists:
                 reset_instrument(session)
 
-            function_enum = str_to_enum(FUNCTION_TO_ENUM, measurement_type)
-            resolution_value = str_to_enum(RESOLUTION_DIGITS_TO_VALUE, str(resolution_digits))
+            function_enum = FUNCTION_TO_DISPLAY_VALUE[measurement_type]
+            resolution_value = RESOLUTION_DIGITS_TO_DISPLAY_VALUE[resolution_digits]
```

Comment:
I reverted to using str_to_enum for resolution_digits in this case so I believe this should no longer be an issue.

It is difficult but not impossible to pass in an invalid enum. If the measurement panel is set up to use an enum control, we do not allow the user to pass in invalid values. However, if the enum configuration can still be bound to a numeric control, then it would be possible. Additionally, if the enum control in the measui file does not match the enum in the service, then it may also be possible to pass in an invalid value. This should all get handled in the validation we do during deserialization though so I don't think it's an issue. Let me know if I'm misinterpreting your concern.

Reply:
This is the only remaining call to `str_to_enum` so I think it's appropriate to remove it from `_helpers.py` and change this example to do the lookup directly.